docs/tuf-spec.md

```diff
@@ -514,6 +519,15 @@ Version: **1.0 (Draft)**
 
    where PUBLIC is a 32-byte string.
 
+   The 'ecdsa' format is:
+
+        { "keytype" : "ecdsa-sha2-nistp256",
+          "scheme" : "ecdsa-sha2-nistp256",
+          "keyval" : { "public" : PUBLIC}
+        }
+
+   where PUBLIC is in PEM format and a string.
+
```

Comment:
I don't see anywhere in the spec where private key formats are explained. Perhaps you should add a note after these examples or something, saying that for a private key, keyval will also contain "private": PRIVATE.

Reply:
We previously included a "private" attribute in KEYVAL, but later decided to remove altogether from metadata.  Since public metadata is never expected to contain a private key, people argued that the "private" attribute shouldn't exist.

Although the repository tool stores private keys in KEYVAL for convenience, it is not required by the specification.  Taking into account that adopters don't have to worry about producing metadata with a "private" element, should we bother explaining it in the specification?